### PR TITLE
task6 

### PR DESCRIPTION
**Pull request by owsym**

### CODE REVIEW

File: task/t6.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from utils.reader import get_file_content
+from datetime import datetime
+file_contents  =get_file_content("/home/owsym/Project/Python/barq_weather_task/files/f2.csv")
```
Comment (Hamma111) [inline task/t6.py:3]: fix the formatting!! formatting should be fixed always

File: task/t6.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+for file_value in file_contents :
+    date = file_value[1]
+    datetime = datetime.strptime(date, "%Y-%m-%d")
+    weekday = datetime.strftime("%A")
+    events = file_value[-2]
```
Comment (Hamma111) [inline task/t6.py:9]: `event` would be a better name since it is only one event

File: task/t6.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+    if events == "Thunderstorm":
+        print(f"This {weekday} , Event IS > {events}")
```
Comment (Hamma111) [inline task/t6.py:12]: new line at the end!

**Review by Hamma111 — CHANGES_REQUESTED** [review]

File: task/t6.py (hunk 1 above)
Comment (owsym) [reply task/t6.py:3]: ok

File: task/t6.py (hunk 2 above)
Comment (owsym) [reply task/t6.py:9]: done

File: task/t6.py (hunk 3 above)
Comment (owsym) [reply task/t6.py:12]: ok

**Pull request merged**